crates/orchestrator/src/lib.rs

```diff
@@ -82,8 +88,36 @@ impl<KEY: SignatureKey + 'static, ELECTION: ElectionConfig + 'static>
             pub_posted: HashSet::new(),
             nodes_connected: 0,
             start: false,
+            bench_results: BenchResults::default(),
+            nodes_post_results: 0,
         }
     }
+
+    /// Output the results to a csv file according to orchestrator state
+    pub fn output_to_csv(&self) {
+        let output_csv = BenchResultsDownloadConfig {
+            commit_sha: self.config.commit_sha.clone(),
+            total_nodes: self.config.config.total_nodes.into(),
+            da_committee_size: self.config.config.da_committee_size,
+            transactions_per_round: self.config.transactions_per_round,
+            transaction_size: self.config.transaction_size,
```

Comment:
Do we need to add 8 bytes here for the timestamp too?

Reply:
Oh good catch! Let me do some changes.

Reply:
Seems like having a lot of `+ 8`'s everywhere could potentially introduce some issues down the road. For example, if I were asked to edit this code, and had not been in this conversation, I may have missed the transaction_size change and introduced a bug here. Is there perhaps a way to signify that transaction_size must ALWAYS have that + 8 for the timestamp? This is a bit of a meta-discussion, so I'm happy to punt this to later, but I thought it might be worth at least documenting (however ephemeral review comments are).

Reply:
Yes! So I did an update on simply changing to `self.bench_results.transaction_size_in_bytes`, and we don't need `+8` elsewhere.